source/math/FloatVectorList.ooc

```diff
@@ -198,4 +198,23 @@ FloatVectorList: class extends VectorList<Float> {
 		forwardGaussian free()
 		result
 	}
+	getWaveletTransform: func (levels: Int) -> VectorList<This> {
+		result := VectorList<This> new(levels)
+		previous := this
+		filtered: This
```

Comment:
OK, this is the C output:
```
    #line 202 "/home/dhesselbom/versioned/ooc/ooc-kean/source/math/FloatVectorList.ooc"
    VectorList__VectorList* result = VectorList__VectorList_new_heap((lang_types__Class*)FloatVectorList__FloatVectorList_class(), levels);
    #line 203 "/home/dhesselbom/versioned/ooc/ooc-kean/source/math/FloatVectorList.ooc"
    FloatVectorList__FloatVectorList* previous = this;
    #line 204 "/home/dhesselbom/versioned/ooc/ooc-kean/source/math/FloatVectorList.ooc"
    FloatVectorList__FloatVectorList* filtered = NULL;
```
So I guess ooc can handle it after all. Still, it's good practice to always initialize your variables before using them.